src/org/omegat/util/gui/UIDesignManager.java

```diff
@@ -69,12 +71,47 @@
  * @author Alex Buloichik
  * @author Yu Tang
  * @author Aaron Madlon-Kay
+ * @author Hiroshi Miura
  */
 public final class UIDesignManager {
 
+    private static final Set<String> BLACKLIST = new HashSet<>();
+
+    static {
+        BLACKLIST.add("CDE/Motif");
+        BLACKLIST.add("Metal");
+        BLACKLIST.add("GTK+");
```

Comment:
Why these items? Are we sure no one wants to use them?

I think some people set the LAF on the command line (see https://docs.oracle.com/javase/tutorial/uiswing/lookandfeel/plaf.html) like

```shell
java -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel MyApp
```

Will that no longer work?

Reply:
OK, I'd like drop black listing default LaFs.
Users always decide by themselves whether use  it or not.

Reply:
> Will that no longer work?

A configuration `java -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel MyApp` override a default `SystemClassLoader` so it working when user set theme as Default.

Reply:
>  OK, I'd like drop black listing default LaFs.

Great, thanks.

> so it working when user set theme as Default

That's great. I'd like to avoid breaking existing setups if possible.